cinch/roles/jenkins_common/tasks/main.yml

```diff
@@ -43,3 +43,10 @@
     owner: "{{ jenkins_user }}"
     mode: 0600
   notify: generate public SSH key
+
+- name: add ssh keys to authorized_keys
```

Comment:
Let's add "for the jenkins user" to the name of this task so in log output it's a bit more clear what this is doing.